java/binary_search.java

```diff
@@ -0,0 +1,44 @@
+import java.util.Scanner;
+
+class BinarySearch
+{
+  public static void main(String args[])

```

Comment:
Add a separate static method binary_search which takes sorted array and an element as argument. 
Add a method check in which you'll add your assertions. Call that test method in main.